include/cpp-statsd-client/UDPSender.hpp

```diff
@@ -118,10 +141,35 @@ class UDPSender final {
 
     //! Error message (optional string)
     std::string m_errorMessage;
+};
+
+#ifdef _WIN32
```

Comment:
in windows you have to initialize and tear down `WSA` before doing socket operations. so i wrapped it in a threadsafe singleton idiom and used the RAII idiom to take care of the teardown